pyproject.toml

```diff
@@ -1,7 +1,93 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools>=67.0.0"]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "pipenv"
+authors = [
+  {name = "Pipenv maintainer team", email = "distutils-sig@python.org"},
+]
+description = "Python Development Workflow for Humans."
+readme = "README.md"
+requires-python = ">=3.7"
+license = {file = "LICENSE"}
+classifiers=[
+  "License :: OSI Approved :: MIT License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = [
+  "certifi",
+  "setuptools>=67.0.0",
+  "virtualenv>=20.24.2",
+]
+dynamic = ["version"]
+
+
+[project.optional-dependencies]
+dev = [
+  "towncrier",
+  "beautifulsoup4",
+  "sphinx",
+  "flake8>=3.3.0,<4.0",
+  "black==23.3.0",
+  "parver",
+  "invoke",
+]
+tests = [
+  "pytest>=5.0",
+  "pytest-timeout",
+  "pytest-xdist",
+  "flaky",
+  "mock",
+]
+
+
+[project.scripts]
+pipenv = "pipenv:cli"
+pipenv-resolver = "pipenv.resolver:main"
+
+
+[project.urls]
+Homepage = "https://github.com/pypa/pipenv"
+Documentation = "https://pipenv.pypa.io/en/latest/"
+Source = "https://github.com/pypa/pipenv.git"
+
+
+[tool.setuptools.packages.find]
+where = ["."]
+exclude = ["tests*", "tests.*", "tasks*", "tasks.*"]
+
+
+[tool.setuptools.package-data]
+"*" = ["LICENSE", "NOTICES"]
+"pipenv.patched.safety" = ["VERSION", "safety-policy-template.yml"]
+"pipenv.patched.pip._vendor.certifi" = ["*.pem"]
+"pipenv.patched.pip._vendor.requests" = ["*.pem"]
+"pipenv.patched.pip._vendor.distlib._backport" = ["sysconfig.cfg"]
```

Comment:
We can remove this line. distlib   no longer includes _backport.

Reply:
@oz123 You okay with changes?